oneflow/api/python/autograd/autograd_function_state.cpp

```diff
@@ -23,6 +23,15 @@ limitations under the License.
 namespace py = pybind11;
 namespace oneflow {
 namespace one {
+namespace {
+inline FunctionAutoGradCaptureState* CheckAndGetStateData(PyAutogradFunctionState* state) {
+  if (!state->data.lock()) {
+    PyErr_Format(PyExc_RuntimeError, "can't assign Tensor as its own grad");
```

Comment:
"data is deallocated. please don't hold context outside autograd.Function.forward or autograd.Function.backward"

Reply:
done